Component/NativeSlugify.php

```diff
@@ -1,50 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sonata package.
- *
- * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\CoreBundle\Component;
-
-/**
- * Do not use this class, it is only here to be BC.
- *
- * @deprecated
```

Comment:
We don't have any deprecation description and no `trigger_error`.

Could you please add some on another PR on the 3.x branch before merging this one? Thanks.